pkgs/development/python-modules/markdown/default.nix

```diff
@@ -9,14 +9,16 @@
 
 buildPythonPackage rec {
   pname = "markdown";
-  version = "3.3.6";
+  version = "3.3.7";
 
   disabled = pythonOlder "3.6";
 
+  format = "setuptools";
```

Comment:
```suggestion
  version = "3.3.7";
  format = "setuptools";

  disabled = pythonOlder "3.6";
```

Reply:
Why?

Reply:
That is the ordering we normally use across many packages but it wasn't important enough to block the merge.